src/main/java/com/glodblock/github/network/SPacketMEFluidInvUpdate.java

```diff
@@ -40,8 +38,12 @@ public SPacketMEFluidInvUpdate(byte b) {
 
     /**
      * If resort, call "updateView()". Used when multiple packets are sent to open an inventory; only the last packet
-     * should resort.
+     * should resort. <<<<<<< HEAD
      *
+     * =======
+     *
+     * >>>>>>> 669eb96... Network throttle: Force ME update packets to be sent once every 50ms.
+     * 
```

Comment:
Is that intended? Looks like a git diff left over.

Reply:
i removed it